pkg/cmd/registry/artifact/crud/get/get.go

```diff
@@ -95,33 +99,61 @@ func runGet(opts *options) error {
 		opts.Logger.Info(opts.localizer.MustLocalize("registry.artifact.common.message.no.group", localize.NewEntry("DefaultArtifactGroup", registrycmdutil.DefaultArtifactGroup)))
 	}
 
-	var dataFile *os.File
-	if opts.version != "" {
-		opts.Logger.Info(opts.localizer.MustLocalize("artifact.common.message.fetching.with.version", localize.NewEntry("Version", opts.version)))
-		request := dataAPI.VersionsApi.GetArtifactVersion(opts.Context, opts.group, opts.artifact, opts.version)
-		dataFile, _, err = request.Execute()
-	} else {
-		opts.Logger.Info(opts.localizer.MustLocalize("artifact.common.message.fetching.latest"))
-		request := dataAPI.ArtifactsApi.GetLatestArtifact(opts.Context, opts.group, opts.artifact)
-		dataFile, _, err = request.Execute()
-	}
-	if err != nil {
-		return registrycmdutil.TransformInstanceError(err)
-	}
-	fileContent, err := os.ReadFile(dataFile.Name())
-	if err != nil {
-		return err
-	}
-	if opts.outputFile != "" {
-		err := os.WriteFile(opts.outputFile, fileContent, 0600)
+	if !opts.references {
+
+		var dataFile *os.File
+		if opts.version != "" {
+			opts.Logger.Info(opts.localizer.MustLocalize("artifact.common.message.fetching.with.version", localize.NewEntry("Version", opts.version)))
+			request := dataAPI.VersionsApi.GetArtifactVersion(opts.Context, opts.group, opts.artifact, opts.version)
+			dataFile, _, err = request.Execute()
+		} else {
+			opts.Logger.Info(opts.localizer.MustLocalize("artifact.common.message.fetching.latest"))
+			request := dataAPI.ArtifactsApi.GetLatestArtifact(opts.Context, opts.group, opts.artifact)
+			dataFile, _, err = request.Execute()
+		}
+		if err != nil {
+			return registrycmdutil.TransformInstanceError(err)
+		}
+		fileContent, err := os.ReadFile(dataFile.Name())
 		if err != nil {
 			return err
 		}
+		if opts.outputFile != "" {
+			err := os.WriteFile(opts.outputFile, fileContent, 0600)
+			if err != nil {
+				return err
+			}
+		} else {
+			// Print to stdout
+			fmt.Fprintf(os.Stdout, "%v\n", string(fileContent))
+		}
+
+		opts.Logger.Info(icon.SuccessPrefix(), opts.localizer.MustLocalize("artifact.common.message.fetched.successfully"))
+
 	} else {
-		// Print to stdout
-		fmt.Fprintf(os.Stdout, "%v\n", string(fileContent))
+		version, err := versionOrLatest(dataAPI, opts)
+		if err != nil {
+			return err
+		}
+		result, _, err := dataAPI.VersionsApi.GetArtifactVersionReferences(opts.Context, opts.group, opts.artifact, *version).Execute()
+		if err != nil {
+			return registrycmdutil.TransformInstanceError(err)
+		}
+		util.PrettyPrintReferences(os.Stdout, result)
```

Comment:
I suggest adding an option to choose the output format like in the other commands.